### DISCUSSION TITLE
Command line arguments?

### DISCUSSION THREAD

**Discussion by jftheoret** [winpython/winpython#1305] [Q&A]
Is it possible to start a batch file when "WinPython Command Prompt.exe" is started (like the /K option for cmd.exe)?

**Comment by stonebig** [comment]
\scripts\env.bat ?

or

\scripts\cmd.bat ?

all icons point to a script in \scripts

**Comment by GuanglinDu** [comment]
@echo off

REM Here's a batch file I use with WinPython to launch my Python application.
REM 1/2. Put this batch file in the WinPython root folder;
REM 2/2. Put example.py in the notebooks folder under WinPython.

REM I based my batch script on the following Stack Overflow answer.
REM Make an batch file that runs commando in WinPython Command Prompt
REM https://stackoverflow.com/questions/31113114/make-an-batch-file-that-runs-commando-in-winpython-command-prompt

SETLOCAL
CD /d "%~dp0"
CALL "scripts\env_for_icons.bat"
CMD /k python.exe example.py
